client/http/client.go

```diff
@@ -464,16 +372,17 @@ func (c *client) request(ctx context.Context, reqInfo *requestInfo, headerOpts .
 		headerOpts...)
 }
 
-// UpdateMembersInfo updates the members info of the PD cluster in the inner client.
-// Exported for testing.
-func (c *client) UpdateMembersInfo() {
-	c.inner.updateMembersInfo(c.inner.ctx)
+// requestChecker is used to check the HTTP request sent by the client.
+type requestChecker func(req *http.Request) error
```

Comment:
cc @HuSharp

Reply:
Is it just used in the test?

Reply:
Is it just used in the test?

Reply:
Yes

Reply:
maybe better to add some comments for `NewHTTPClientWithRequestChecker`